src/init/common.cpp

```diff
@@ -125,8 +125,9 @@ bool StartLogging(const ArgsManager& args)
     if (fs::exists(config_file_path)) {
         LogPrintf("Config file: %s\n", fs::PathToString(config_file_path));
     } else if (args.IsArgSet("-conf")) {
```

Comment:
given that checking the args is not for free (triggers locks and merges from multiple sources), maybe we could introduce a `IsArgEnabled` which gets rid of this confusing combination (where `-noconf=0` would be set and not negated, I think)